sample_underwriting.yml

```diff
@@ -0,0 +1,28 @@
+---
+# sample underwriting format
+- name: back to school
+  startdate: 2017-07-03
+  enddate: 2017-08-20
+  sponsor: nrv mall
+  transcript: --- >
+    I'm going to put a sample fake transcript into this for
+    fun because it could be useful
+  url:
+  - 'https://example.com/underwiting/today.mp3'
+  - 'https://example.com/underwiting/tomorrow.flac'
+  schedule:
+    monday: [0,1,2,3,4,5]
+    tuesday: [0,3,4,23,8]
+- name: end of world
+  startdate: 2012-01-01
+  enddate: 2018-03-04
+  sponsor: nrv mall
+  url: [ /underwriting/the_day_after.ogg, /dev/urandom ]
```

Comment:
`/dev/urandom` is not a good URL to use.

Reply:
it's an arbitrary example to test parsing. I don't see why there's an issue